src/snowflake/cli/api/project/schemas/native_app/native_app.py

```diff
@@ -34,6 +34,10 @@ class NativeApp(UpdatableModel):
     artifacts: List[Union[PathMapping, str]] = Field(
         title="List of file source and destination pairs to add to the deploy root",
     )
+    bundle_root: Optional[str] = Field(
```

Comment:
I'm not 100% sold on bundle root, but I can't think of a better name right now...